adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationParamsTests.java

```diff
@@ -151,6 +152,7 @@ public void testCreateFromResourceUrlPositive() throws IOException {
                 .getAuthority().trim());
     }
 
+    @Ignore
```

Comment:
ADO task?

Reply:
I'll revisit each of the repos once this is done and create a work item for each repo relative to ignored tests.